android/build.gradle

```diff
@@ -349,7 +347,7 @@ android {
       resValue 'string', 'app_id', android.defaultConfig.applicationId + applicationIdSuffix
       resValue 'string', 'app_name', project.ext.appName + ' ' + '(Beta)'
       ndk {
-        debugSymbolLevel 'SYMBOL_TABLE'
+        debugSymbolLevel 'FULL'
```

Comment:
I tried this option before and Google Play didn't accept the bundle because of size.

Reply:
Thanks, I didn't know it.